app/_hub/kong-inc/session/_index.md

```diff
@@ -72,6 +147,20 @@ params:
       datatype: string
       description: The domain with which the cookie is intended to be exchanged.
     - name: cookie_samesite
+      maximum_version: "3.1.x"
+      required: false
+      default: Strict
+      datatype: string
+      description: |
+        Determines whether and how a cookie may be sent with cross-site requests. `Strict`:
+        The browser will send cookies only if the request originated from the website that set the cookie.
+        `Lax`: Same-site cookies are withheld on cross-domain subrequests, but will be sent when a user navigates
+        to the URL from an external site, for example, by following a link. `None` or `off`: Disables
+        the same-site attribute so that a cookie may be sent with cross-site requests. `None` requires
+        the Secure attribute (`cookie_secure`) in latest browser versions. For more info, see the
+        [SameSite cookies docs on MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite).'
```

Comment:
```suggestion
        Determines whether and how a cookie may be sent with cross-site requests. 
        
        * `Strict`: The browser sends cookies only if the request originated from the website that set the cookie.
        * `Lax`: Same-site cookies are withheld on cross-domain subrequests, but are sent when a user navigates
        to the URL from an external site, for example, by following a link. 
        * `None` or `off`: Disables the same-site attribute so that a cookie may be sent with cross-site requests. 
        `None` requires the Secure attribute (`cookie_secure`) in latest browser versions. For more information, 
        see the [SameSite cookies docs on MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite).'
```